courier/smtp.go

```diff
@@ -28,6 +28,20 @@ type smtpClient struct {
 
 func newSMTP(ctx context.Context, deps Dependencies) *smtpClient {
 	uri := deps.CourierConfig(ctx).CourierSMTPURL()
+	var tlsCertificates []tls.Certificate
+	clientCertPath := deps.CourierConfig(ctx).CourierSMTPClientCertPath()
+	clientKeyPath := deps.CourierConfig(ctx).CourierSMTPClientKeyPath()
+
+	if clientCertPath != "" && clientKeyPath != "" {
```

Comment:
Would it be possible to provide a test for this?

Reply:
Hi! I added some unit test